api/kyverno/v2alpha1/policy_exception_types.go

```diff
@@ -20,26 +20,17 @@ import (
 
 	kyvernov2beta1 "github.com/kyverno/kyverno/api/kyverno/v2beta1"
 	"github.com/kyverno/kyverno/pkg/engine/variables/regex"
-	"github.com/kyverno/kyverno/pkg/utils/wildcard"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 )
 
 // +genclient
 // +kubebuilder:object:root=true
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
-// +kubebuilder:object:root=true
```

Comment:
@MariamFahmy98 don't we need to keep this ?

Reply:
It is duplicated at lines 29 and 31. I removed it from line 31.
https://github.com/kyverno/kyverno/blob/86530740c10ebe976865d98517ac30fbb998e75c/api/kyverno/v2alpha1/policy_exception_types.go#L29-L31

Reply:
🙈